src/utils/validations.js

```diff
@@ -6,11 +6,11 @@ export const validateEmail = (email) => {
   }
 
   if (!email.includes("@")) {
-    return "Email must contain '@'.";
+    return "Email must contain '@'";
   }
 
   if (!email.includes(".")) {
-    return "Email must contain a '.'.";
+    return "Email must contain a '.'";
   }
 
```

Comment:
I think it would be better for email to show "Email must be a valid email address"

Reply:
I just deleted the dot that's olaoluwa's problem how he implemented the email check

Reply:
Is there a ticket to address this?